browser-test/src/end_to_end_enumerators.test.ts

```diff
@@ -1,54 +1,432 @@
-import {test, expect} from '@playwright/test'
+import {test, expect} from './support/civiform_fixtures'
 import {
   AdminPrograms,
   AdminQuestions,
-  createTestContext,
+  enableFeatureFlag,
   loginAsAdmin,
   logout,
   validateAccessibility,
   validateScreenshot,
   waitForPageJsLoad,
 } from './support'
+import {Page} from 'playwright'
 
 test.describe('End to end enumerator test', () => {
   const programName = 'Ete enumerator program'
-  const ctx = createTestContext(/* clearDb= */ false)
 
-  test('Updates enumerator elements in preview', async () => {
-    const {page, adminQuestions} = ctx
-    await loginAsAdmin(page)
+  test.describe('Admin page', () => {
```

Comment:
**suggestion:** We can add the uses-fixures tag here https://github.com/civiform/civiform/pull/6984/files#r1532474323

Reply:
More info if needed https://github.com/civiform/civiform/wiki/Playwright-Feature-Migration-Guide